### ISSUE TITLE
Allow scrolling on click and drag

### ISSUE THREAD

**Issue by Thyverity** [suchnsuch/Tangent#71]

**Comment by taylorhadden** [comment]
The side bar should automatically scroll if you move your mouse to the top/bottom of the sidebar. Is that working for you?

**Comment by Thyverity** [comment]
No.. The side bar doesn't scroll on my end. I have a dual boot install, will test to see if it works on Windows.

**Comment by Thyverity** [comment]
So this is embarrassing, it is working fine, for some reason I couldn't do it before and I don't know why.

**Comment by Thyverity** [comment]
Could you increase the detection area when dragging?

**Comment by taylorhadden** [comment]
Not sure. That scroll-on-drag behavior is raw chromium.
